pages.ja/common/docker.md

```diff
@@ -0,0 +1,37 @@
+# docker
+
+> Dockerコンテナ及びDockerイメージの管理を行います。
+> `docker run`などいくつかのサブコマンドがあります。これらには使用方法についての独自のドキュメントがあります。
+> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/cli/>
+
+- 全てのDockerコンテナを表示する(実行中・停止中、両方のコンテナ):
+
+`docker ps --all`
+
+- イメージからカスタムのコンテナ名でコンテナを起動する:
+
+`docker run --name {{コンテナ名}} {{イメージ}}`
+
+- 既存のコンテナの起動もしくは停止を行う:
+
+`docker {{start|stop}} {{コンテナ名}}`
+
+- dockerレジストリからイメージをプルする
```

Comment:
```suggestion
- dockerレジストリからイメージをプルする:
```

Reply:
I overlooked it! I'll respond with additional commits.

Reply:
Fixed with the following additional commitments!

https://github.com/tldr-pages/tldr/pull/9785/commits/98edc03e12aed1a91e7b2f13ae7a4b248b38269c